test/test_helper.rb

```diff
@@ -2,10 +2,12 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
 require 'mocha/mini_test'
+require 'capybara/rails'
```

Comment:
@tgxworld  we require it in Integration tests(current test) as well, Shall I include it only with that test ?